pkg/codefresh/argo_runtime.go

```diff
@@ -0,0 +1,82 @@
+package codefresh
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/codefresh-io/argo-platform/libs/ql/graph/model"
+)
+
+
+type (
+	IArgoRuntimeAPI interface {
+		List() ([]model.Runtime, error)
+	}
+	argoRuntime struct {
+		codefresh *codefresh
+	}
+	graphqlRuntimesResponse struct {
+		Data graphqlRuntimesInnerResponse`json:"data"`
+	//	Errors []byte
```

Comment:
Is there an easy way to simulate an error response from the platform, to see how the client should handle it?